pkg/config/config_test.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"testing"
+	"time"
+)
+
+func TestConfig_GetShareRelistInterval(t *testing.T) {
```

Comment:
OK @otaviof wrt the need for an e2e in addition to your unit tests here, I think it is valid to have tests verify that 
1) our daemonset pod's exit successfully and k8s restarts our pods
2) say you add a new namespace to ignore; you could have pods that were provisioned with read/write volumes to a secret or configmap from that namespace, and prove they see updates to the config map or secret; you then change the config to start ignoring that namespace; the driver restarts; you then can validate that the pod no longer sees additional updates from the config map or secret from the namespace that is now ignored  .... I could envision a cluster admin wanting to do this as a means of "freezing" the content in question from consuming pods, or other scenarios we have not envisioned yet .... it is not just that the setting in the config file is verified as changed.  It is the propagation of the change end to end.  And yes, we have tests to verify those specific settings.  But one of the common reasons for e2e testing is to validate that multiple features work in conjunction.
3) the same scenario could be augmented to a global level, and we freeze updates everywhere by setting refresh resources to false

I'd be curious if @adambkaplan or @coreydaley either a) feel I asking more than what should be minimally expected for this story.  If I'm the sole opinion who things such e2e's are a good idea, I'll defer.  I'm also curious if either of them can think of other e2e type scenarios with respect to config change that I am omitting here.

Reply:
But I don't see this existing unit test covering any of ^^ above